core/embed/rust/src/ui/display/font.rs

```diff
@@ -248,14 +249,33 @@ impl Font {
         (start + end + self.visible_text_height(text)) / 2
     }
 
+    fn get_glyph_data(&self, c: u16) -> *const u8 {
```

Comment:
How about renaming `c` to `codepoint` here?

Reply:
45d747fa91f2d5e8095a70708a124de6ca69186b

but I'll improve the function using matejcik points